Library/Formula/xorg.rb

```diff
@@ -0,0 +1,470 @@
+class Xorg < Formula
+  desc "Xorg Libraries"
+  homepage "http://www.linuxfromscratch.org/blfs/view/svn/x/x7lib.html"
```

Comment:
Strictly a matter of opinion, but I think http://www.x.org/wiki/ would make more sense for the homepage.